docs/en/sql-reference/aggregate-functions/reference/covarpop.md

```diff
@@ -1,14 +1,56 @@
 ---
 slug: /en/sql-reference/aggregate-functions/reference/covarpop
-sidebar_position: 36
+sidebar_position: 37
 ---
 
 # covarPop
 
-Syntax: `covarPop(x, y)`
-
 Calculates the value of `Σ((x - x̅)(y - y̅)) / n`.
 
 :::note
 This function uses a numerically unstable algorithm. If you need [numerical stability](https://en.wikipedia.org/wiki/Numerical_stability) in calculations, use the `covarPopStable` function. It works slower but provides a lower computational error.
-:::
\ No newline at end of file
+:::
```

Comment:
I guess we should mention the term "population covariance" somewhere in l. 8 (instead of only listing a formula).